units/XNL0401/XNL0401_Script.lua

```diff
@@ -169,11 +180,14 @@ XNL0401 = Class(NExperimentalHoverLandUnit, SlowHover) {
         end
     end,
 
+    ---@param self XNL0401 unused
+    ---@return boolean
     CrushDuringDescent = function(self)
         -- This is a WOF event which is in here cause I like WOF! Sorry, I can't be crushed during sinking!
         return false
     end,
 
+    ---@param self XNL0401
     OnSeabedImpact = function( self)
```

Comment:
There are many similar whitespace errors in the file.
```suggestion
    OnSeabedImpact = function(self)
```